subsys/bluetooth/controller/Kconfig

```diff
@@ -306,7 +307,7 @@ config BT_CTLR_DATA_LENGTH
 	# Update procedure in the Controller.
 	bool
 	depends on BT_DATA_LEN_UPDATE
-	default y if SOC_COMPATIBLE_NRF52X || BT_CTLR_DATA_LENGTH_CLEAR
+	default y if (SOC_COMPATIBLE_NRF52X || SOC_SERIES_NRF53X) || BT_CTLR_DATA_LENGTH_CLEAR
```

Comment:
Heads up about conflicting changes here:  https://github.com/zephyrproject-rtos/zephyr/pull/19710#discussion_r334458656

Reply:
Thanks. For now I think we can live with this, but I agree we should try to have a "capability" Kconfig symbol for Data Length Extension 
I'll make a note to file an issue for that.

Reply:
@ioannisg They are already doing that change. So we just have to use it once it is merged. It should make this a lot easier.